.github/workflows/build.yml

```
@@ -16,12 +16,12 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 8.0]
-                symfony: [^4.4, ^5.2]
+                php: ["7.4", "8.0", "8.1"]
+                symfony: ["^4.4", "^5.2"]
```

I wonder should we test it only with 5.4
```suggestion
                symfony: ["^4.4", "^5.4"]
```